fulp_modules/features/toys/code/plushies.dm

```diff
@@ -93,8 +93,29 @@
 	desc = "Made from real meat!"
 	icon = 'fulp_modules/icons/toys/toys.dmi'
 	icon_state = "beefman"
+	base_icon_state = "beefman"
 	squeak_override = list('sound/effects/meatslap.ogg'=1)
 
+	/// If true then the plushie will initialize with its holiday icon (which has a santa hat).
+	/// Coded by referencing 'supermatter.dm'
+	var/holiday_edition = FALSE
+
+/obj/item/toy/plush/beefplushie/Initialize(mapload)
+	. = ..()
+	if(!isnull(check_holidays(FESTIVE_SEASON)))
+		holiday_edition = TRUE
+		update_appearance()
+
+/obj/item/toy/plush/beefplushie/update_desc(mob/user)
+	. = ..()
+	if(holiday_edition)
+		desc += span_notice("\nIt's a holiday exclusive edition with a santa hat sewn on!")
```

Comment:
```suggestion
		desc = initial(desc) + span_notice("\nIt's a holiday exclusive edition with a santa hat sewn on!")
```
in case update appearance is called multiple times, this would actually stack.

Reply:
Ah— I would have never considered that.